client/src/components/SectionTable/SectionTable.js

```diff
@@ -44,6 +44,7 @@ const styles = {
 class SectionTable extends PureComponent {
     render() {
         const { classes } = this.props;
+        const urlEncode = encodeURIComponent(this.props.courseDetails.deptCode);
```

Comment:
We can do some  _fancy_  [Javascript Destructuring](https://javascript.info/destructuring-assignment#object-destructuring) here! 

Adding `courseDetails` on line 46 will pull the variable out of `this.props`, so it's easier to reference later on.
So on line 47 (and 97), we can remove `this.props`.

```suggestion
        const { classes, courseDetails } = this.props;
        const urlEncode = encodeURIComponent(courseDetails.deptCode);
```